galicaster/plugins/keyboard.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding:utf-8 -*-
+# Galicaster, Multistream Recorder and Player
+#
+#       galicaster/plugins/keyboard
+#
+# Copyright (c) 2012, Teltek Video Research <galicaster@teltek.es>
+#
+# This work is licensed under the Creative Commons Attribution-
+# NonCommercial-ShareAlike 3.0 Unported License. To view a copy of 
+# this license, visit http://creativecommons.org/licenses/by-nc-sa/3.0/ 
+# or send a letter to Creative Commons, 171 Second Street, Suite 300, 
+# San Francisco, California, 94105, USA.
+
+"""
+Plugin that shows a keyboard when a dialog window with input entries is shown
+"""
+
+from gi.repository import Gtk, Gdk
+from galicaster.core import context
+import subprocess
+import re
+
+from galicaster.utils.i18n import _
+
+logger = context.get_logger()
+pid = None
+
+def init():
+    global pid
+    dispatcher = context.get_dispatcher()
+
+    pid = is_running('onboard')
+    if not pid:
+        pid = subprocess.Popen(["onboard"]).pid
+
+    dispatcher.connect('init', configure_keyboard)
+    dispatcher.connect('quit', unconfigure_keyboard)
+
+def configure_keyboard(dispatcher=None):
+    execute(['gsettings',
+        'set',
+        'org.onboard.auto-show',
+        'hide-on-key-press',
+        'false'])
+
+    execute(['gsettings',
+        'set',
+        'org.onboard.auto-show',
+        'enabled',
+        'true'])
+
+    execute(['gsettings',
+        'set',
+        'org.onboard',
+        'layout',
+        'Phone'])
+
+    execute(['gsettings',
+        'set',
+        'org.onboard',
+        'theme',
+        'Ambiance'])
+
+    execute(['gsettings',
+        'set',
+        'org.onboard.theme-settings',
+        'color-scheme',
+        '/usr/share/onboard/themes/Aubergine.colors'])
+
+    execute(['gsettings',
+        'set',
+        'org.onboard',
+        'start-minimized',
+        'true'])
+
+    execute(['gsettings',
+        'set',
+        'org.onboard.window.landscape',
+        'dock-expand',
+        'false'])
+
+
+def unconfigure_keyboard(dispatcher=None):
+    execute(['gsettings',
+        'set',
+        'org.onboard',
+        'use-system-defaults',
+        'true'])
+
+    execute(['kill', str(pid)])
```

Comment:
Why use a global var for PID? Getting the PID in the `unconfigure_keyboard` function looks better.